### DISCUSSION TITLE
Extend RTL to work with "Google Chrome Translate this page" option

### DISCUSSION THREAD

**Discussion by jabranr** [tailwindlabs/tailwindcss#12919] [Help]
Google Chrome's context menu option "Translate this page to ..." does not add the `dir=rtl` attribute to `<html>` tag. Therefore the tailwind classes for `rtl-*`  do not work as expected. Google Chrome however adds a class `translated-rtl` to the `<html>` tag. I am wondering if I can extend the core `rtl-*` somehow to add CSS match for this i.e.

**Current**
```css
[dir=rtl] .rtl:name { ... }
```
**Expected**
```css
[dir=rtl] .rtl:name, [class*=rtl] .rtl:name { ... }
```

Appreciate any ideas for it.

**Comment by wongjn** [comment]
You could consider a custom variant:

```js
addVariant('foo', '&:where([dir="rtl"], [dir="rtl"] *, [class="translated-rtl"], [class="translated-rtl"] *)');
```

https://play.tailwindcss.com/rGJWqADUUL?file=config
